lib/paratii.eth.js

```diff
@@ -67,13 +68,36 @@ export class ParatiiEth {
       throw Error(msg)
     }
     let contract = this.contracts[name]
+
     let deployedContract = await contract.deploy({arguments: args}).send()
+
     return deployedContract
   }
+  async sleep (ms) {
+    return new Promise(resolve => setTimeout(resolve, ms))
+  }
+
+  async fixMethodAndSend (method, opts) {
+    // this methos fix a know bug on web3, apparently, 'deployContract(...)' does not return the same object as `getContract()` [<- this is a bug]
+    let rawTransaction = await method
+    rawTransaction._ethAccounts = this.web3.eth.accounts
+    // wait for receipt let nonce increment
+    await rawTransaction.send(opts)
+  }
+
+  async fixMethodAndCall (method) {
+    // this methos fix a know bug on web3, apparently, 'deployContract(...)' does not return the same object as `getContract()` [<- this is a bug]
```

Comment:
I would suspect that `call`ing methods works also withtout having hte accounts set - i.e. that for `call`s, this patch is not necessary